content/en/synthetics/settings/_index.md

```diff
@@ -35,13 +35,77 @@ further_reading:
 * [Mobile Applications Settings][18]
 
+## Default settings
+
+### Enforced tags settings
+
+Selecting this option ensures that configured tags from the Usage Attribution tab are required by users creating and editing Synthetic tests. They won't be able to save tests without entering all required tags.
+
+Usage Attribution tags allow you to break down cost and usage attributes of this feature by services, applications, or teams. 
+
+### Default locations
+
+Choose the default locations for your [API test][4], [multistep API test][5], or [browser test][6] details. 
+
+Your options include all of the available managed locations Datadog offers and the private locations you set up for your account.
+
+When you are done selecting locations, click **Save Default Locations**.
+
+### Default browsers and devices
+
+Choose the default browser and device types for your [browser test][6] details.
+
+Your options for browsers include Google Chrome, Firefox, and Microsoft Edge. Your options for devices include a large laptop, a tablet, and a small mobile device.
+
+When you are done selecting browsers and devices, click **Save Default Browsers & Devices**.
+
+### Default tags
+
+Choose or add the default tags for your [API test][4], [multistep API test][5], or [browser test][6] details.
+
+When you are done selecting related tags, click **Save Default Tags**.
+
+### Permissions
+
+By default, only users with the [Datadog Admin and Datadog Standard roles][11] can access the Synthetic Monitoring **Default Settings** page. To get access to the **Default Settings** page, upgrade your user to one of those two [default roles][11]. 
+
+If you are using the [custom role feature][12], add your user to any custom role that includes `synthetics_default_settings_read` and `synthetics_default_settings_write` permissions. 
+
+## Integration settings
+
+{{< img src="synthetics/settings/integration_settings.png" alt="Integration Settings page" style="width:100%;">}}
+
+### APM integration for browser tests
+
+Allow URLs to add APM integration headers to those URLs. Datadog's APM integration headers allow Datadog to link browser tests with APM. 
+
+Define which endpoints you want to send the APM headers to by entering a URL in the **Value** field. If the endpoint is being traced and is allowed, your browser test results are automatically tied to its corresponding trace.
+
+Use `*` to allow wider domain names. For example, adding `https://*.datadoghq.com/*` allows everything on `https://datadoghq.com/`. When you are done adding URLs, click **Save APM Integration Settings**. 
+
+For more information, see [Connect Synthetics and APM Traces][15].
+
+### Synthetic data collection and RUM applications
+
+To allow Datadog to collect RUM data from your test runs, click **Enable Synthetic RUM data collection**. If disabled, you cannot edit the RUM setting in the browser test recorder. When you are done enabling data collection, click **Save RUM Data Collection**.
```

Comment:
```suggestion
To allow Datadog to collect RUM data from your test runs, click **Enable Synthetic RUM data collection**. If disabled, you cannot edit RUM settings in the browser test recorder. To apply changes, click **Save RUM Data Collection**.
```

"When you are done enabling data collection" is misleading, I think; I would interpret it as "when you no longer want to enable data collection". I believe the intended meaning is "when you're done clicking the toggle", and have updated the copy to make that clear.